web_x2many_delete_all/static/src/js/web_x2many_delete_all.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2016 Onestein
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_x2many_delete_all.Mixin', function (require) {
+    "use strict";
+    var core = require('web.core');
+    var _t = core._t;
+    var ListRenderer = require('web.ListRenderer');
+
+    ListRenderer.include({
+        events: _.extend({}, ListRenderer.prototype.events, {
+            'click thead .o_list_record_delete_all': '_onTrashAllIconClick',
```

Comment:
the event is hooked to the TH not the button: is it correct?

Reply:
yes you are right !